Samples/Samples/Samples.Shared/DynamicMap_FeaturesPage.xaml

```diff
@@ -23,6 +23,16 @@
 
             <converters:FromNullableBoolToVisibilityConverter x:Key="IsTrueToCollapsed"
 															  VisibilityIfTrue="Collapsed" />
+            <DataTemplate x:Key="ClusterImageTemplate">
+                <Grid Background="Red"
+                      Width="20"
+                      Height="20">
+                    <TextBlock Text="Bonjour"
```

Comment:
What is the bonjour textblock for?

Reply:
Testing DataTemplate for styling. I can Remove it !